mods/ra/maps/soviet-02b/soviet02b.lua

```diff
@@ -0,0 +1,182 @@
+IntroAttackers = { IntroSoldier1, IntroSoldier2, IntroSoldier3 }
+
+BridgeShroudTrigger = { CPos.New(63, 71), CPos.New(64, 71), CPos.New(65, 71), CPos.New(69, 65), CPos.New(70, 65), CPos.New(71, 65) }
+BridgeExplosionTrigger = { CPos.New(66, 69), CPos.New(67, 69), CPos.New(68, 69) }
+TransportTrigger = { CPos.New(75, 58) }
+EnemyBaseShroudTrigger = { CPos.New(64, 52), CPos.New(64, 53), CPos.New(64, 54), CPos.New(64, 55), CPos.New(64, 56), CPos.New(64, 57), CPos.New(64, 58), CPos.New(64, 59), CPos.New(64, 60), CPos.New(64, 61), CPos.New(64, 62), CPos.New(64, 63), CPos.New(64, 64) }
+ParachuteTrigger = { CPos.New(80, 66), CPos.New(81, 66), CPos.New(82, 66), CPos.New(83, 66), CPos.New(84, 66), CPos.New(85, 66),CPos.New(86, 66), CPos.New(87, 66), CPos.New(88, 66), CPos.New(89, 66) }
+EnemyBaseEntranceShroudTrigger = { CPos.New(80, 73), CPos.New(81, 73), CPos.New(82, 73), CPos.New(83, 73), CPos.New(84, 73), CPos.New(85, 73),CPos.New(86, 73), CPos.New(87, 73), CPos.New(88, 73), CPos.New(89, 73) }
+
+SendUSSRParadrops = function()
+	paraproxy1 = Actor.Create("powerproxy.paratroopers", false, { Owner = player })
+	paraproxy1.SendParatroopers(ParachuteBaseEntrance.CenterPosition, false,  Facing.North)
+	paraproxy1.Destroy()
+end
+
+SendUSSRParadropsBase = function()
+	paraproxy2 = Actor.Create("powerproxy.paratroopers2", false, { Owner = player })
+	paraproxy2.SendParatroopers(ParachuteBase1.CenterPosition, false, Facing.East)
+	paraproxy2.Destroy()
+	paraproxy3 = Actor.Create("powerproxy.paratroopers3", false, { Owner = player })
+	paraproxy3.SendParatroopers(ParachuteBase2.CenterPosition, false, Facing.East)
+	paraproxy3.Destroy()
+end
+
+Trigger.OnEnteredFootprint(BridgeShroudTrigger, function(a, id)
+	if not bridgeShroudTrigger and a.Owner == player then
+		bridgeShroudTrigger = true
+		cameraBridge = Actor.Create("camera", true, { Owner = player, Location = CameraBridge.Location })
+		Trigger.AfterDelay(DateTime.Seconds(15), function()
+			if not cameraBridge.IsDead then
+				cameraBridge.Kill()
+			end
+		end)
+	end
+end)
+
+Trigger.OnEnteredFootprint(BridgeExplosionTrigger, function(a, id)
+	if not bridgeExplosionTrigger and a.Owner == player then
+		bridgeExplosionTrigger = true
+		if not BarrelBridge.IsDead then
+			BarrelBridge.Kill()
+		end
+	end
+end)
+
+Trigger.OnEnteredFootprint(EnemyBaseEntranceShroudTrigger, function(a, id)
+	if not enemyBaseEntranceShroudTrigger and a.Owner == player then
+		enemyBaseEntranceShroudTrigger = true
+		cameraBaseEntrance = Actor.Create("camera", true, { Owner = player, Location = CameraBaseEntrance.Location })
+		Trigger.AfterDelay(DateTime.Seconds(15), function()
+			if not cameraBridge.IsDead then
```

Comment:
I also don't think we need that check here, as the camera won't be destroyed somewhere else.
Same for below.